src/finalisers/es.ts

```diff
@@ -35,16 +35,27 @@ export default function es (chunk: Chunk, magicString: MagicStringBundle, { getP
 			}
 		}
 		if (reexports) {
+			if (imports)
+				output += '\n';
 			const starExport = reexports.find(specifier => specifier.reexported === '*');
+			const nsReexport = reexports.find(specifier => specifier.imported === '*' && specifier.reexported !== '*');
```

Comment:
I would love a longer name here, e.g. `nonStarReexport` as with the shortened name, you have no chance of knowing what it means further down without reading the declaration.